sdks/python/apache_beam/examples/snippets/transforms/elementwise/runinference.py

```diff
@@ -0,0 +1,157 @@
+
+    def forward(self, x):
+      out = self.linear(x)
+      return out
+
+  model_state_dict_path = 'gs://apache-beam-samples/run_inference/five_times_table_torch.pt'  # pylint: disable=line-too-long
+  model_class = LinearRegression
+  model_params = {'input_dim': 1, 'output_dim': 1}
+  model_handler = PytorchModelHandlerTensor(
+      model_class=model_class,
+      model_params=model_params,
+      state_dict_path=model_state_dict_path)
+
+  unkeyed_data = numpy.array([10, 40, 60, 90],
+                             dtype=numpy.float32).reshape(-1, 1)
+
+  with beam.Pipeline() as p:
+    predictions = (
+        p
+        | 'InputData' >> beam.Create(unkeyed_data)
+        | 'ConvertNumpyToTensor' >> beam.Map(torch.Tensor)
+        | 'PytorchRunInference' >> RunInference(model_handler=model_handler)
+        | beam.Map(print))
+    # [END torch_unkeyed_model_handler]
+    if test:
+      test(predictions)
+
+
+def torch_keyed_model_handler(test=None):
+    # [START torch_keyed_model_handler]
+  import apache_beam as beam
+  import torch
+  from apache_beam.ml.inference.base import KeyedModelHandler
+  from apache_beam.ml.inference.base import RunInference
+  from apache_beam.ml.inference.pytorch_inference import PytorchModelHandlerTensor
+
+  class LinearRegression(torch.nn.Module):
+    def __init__(self, input_dim=1, output_dim=1):
+      super().__init__()
+      self.linear = torch.nn.Linear(input_dim, output_dim)
+
+    def forward(self, x):
+      out = self.linear(x)
+      return out
+
+  model_state_dict_path = 'gs://apache-beam-samples/run_inference/five_times_table_torch.pt'  # pylint: disable=line-too-long
+  model_class = LinearRegression
+  model_params = {'input_dim': 1, 'output_dim': 1}
+  keyed_model_handler = KeyedModelHandler(
+      PytorchModelHandlerTensor(
+          model_class=model_class,
+          model_params=model_params,
+          state_dict_path=model_state_dict_path))
+
+  keyed_data = [("first_question", 105.00), ("second_question", 108.00),
+                ("third_question", 1000.00), ("fourth_question", 1013.00)]
+
+  with beam.Pipeline() as p:
+    predictions = (
+        p
+        | 'KeyedInputData' >> beam.Create(keyed_data)
+        | "ConvertIntToTensor" >>
+        beam.Map(lambda x: (x[0], torch.Tensor([x[1]])))
+        | 'PytorchRunInference' >>
+        RunInference(model_handler=keyed_model_handler)
+        | beam.Map(print))
+    # [END torch_keyed_model_handler]
+    if test:
+      test(predictions)
+
+
+def sklearn_unkeyed_model_handler(test=None):
+    # [START sklearn_unkeyed_model_handler]
+  import apache_beam as beam
+  import numpy
+  from apache_beam.ml.inference.base import RunInference
+  from apache_beam.ml.inference.sklearn_inference import ModelFileType
+  from apache_beam.ml.inference.sklearn_inference import SklearnModelHandlerNumpy
+
+  sklearn_model_filename = 'gs://apache-beam-samples/run_inference/five_times_table_sklearn.pkl'  # pylint: disable=line-too-long
```

Comment:
Can't find this filesystem in the unit tests. Can you double check this @AnandInguva

Reply:
Do we need `FileSystems` here?

Reply:
I don't think so. Other pytorch examples don't need to import it.

Do we need to install `apache-beam[gcp]`?

Reply:
`Unable to get filesystem from specified path, please use the correct path or ensure the required dependency is installed, e.g., pip install apache-beam[gcp].`

Reply:
We can add a check to skip if GCP is not detected?

Reply:
Yep 
```
try:
  from apache_beam.io.gcp.gcsfilesystem import GCSFileSystem
except ImportError:
  GCSFileSystem = None  # type: ignore
```

Something like this?

Reply:
Since SKlearn is a installed as a beam dependency in each tox environment, we need to add if GCP is installed as well in every to test

Reply:
How do we skip if apache_beam[gcp] is not installed? @tvalentyn @yeandy

Reply:
Typically our tests do sth like:

```
try: 
  from apache_beam.io.gcp.gcsfilesystem import GCSFileSystem
except ImportError: 
  GCSFileSystem = None # type: ignore
...
@unittest.skipIf(gcsfilesystem is None, 'GCP dependencies are not installed')
```

does this approach work here?

Reply:
> Since SKlearn is a installed as a beam dependency in each tox environment, we need to add if GCP is installed as well in every tox test for the Sklearn tests

I may be missing some context here but can we install gcp dependenices for tox environment for sklearn tests similar to https://github.com/apache/beam/blob/64bcc7df2b86b9a531591a5f907e245f1af7998d/sdks/python/tox.ini#L274 ?

Reply:
I figured it out. Thanks